src/file_info.ts

```diff
@@ -255,7 +255,6 @@ export class ExtractionInfo{
     public load(path: string): number{
         if(!fs.existsSync(path)){
             //Doesn't exist
-            vscode.window.showErrorMessage("File at " + path + " does not exist.");
```

Comment:
for future reference, we should only use this sort of error message for things that the user will care about.